sessionctx/variable/varsutil_test.go

```diff
@@ -254,25 +260,30 @@ func (s *testVarsutilSuite) TestVarsutil(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(val, Equals, string(bVal))
 
-	SetSessionSystemVar(v, TiDBEnableStreaming, types.NewStringDatum("1"))
+	err = SetSessionSystemVar(v, TiDBEnableStreaming, types.NewStringDatum("1"))
+	c.Assert(err, IsNil)
 	val, err = GetSessionSystemVar(v, TiDBEnableStreaming)
 	c.Assert(err, IsNil)
 	c.Assert(val, Equals, "ON")
 	c.Assert(v.EnableStreaming, Equals, true)
-	SetSessionSystemVar(v, TiDBEnableStreaming, types.NewStringDatum("0"))
+	err = SetSessionSystemVar(v, TiDBEnableStreaming, types.NewStringDatum("0"))
+	c.Assert(err, IsNil)
 	val, err = GetSessionSystemVar(v, TiDBEnableStreaming)
 	c.Assert(err, IsNil)
 	c.Assert(val, Equals, "OFF")
 	c.Assert(v.EnableStreaming, Equals, false)
 
 	c.Assert(v.OptimizerSelectivityLevel, Equals, DefTiDBOptimizerSelectivityLevel)
-	SetSessionSystemVar(v, TiDBOptimizerSelectivityLevel, types.NewIntDatum(1))
+	err = SetSessionSystemVar(v, TiDBOptimizerSelectivityLevel, types.NewIntDatum(1))
+	c.Assert(err, IsNil)
 	c.Assert(v.OptimizerSelectivityLevel, Equals, 1)
 
 	err = SetSessionSystemVar(v, TiDBDDLReorgWorkerCount, types.NewIntDatum(-1))
 	c.Assert(terror.ErrorEqual(err, ErrWrongValueForVar), IsTrue)
 
-	SetSessionSystemVar(v, TiDBDDLReorgWorkerCount, types.NewIntDatum(int64(maxDDLReorgWorkerCount)+1))
+	er := SetSessionSystemVar(v, TiDBDDLReorgWorkerCount, types.NewIntDatum(int64(maxDDLReorgWorkerCount)+1))
+	c.Assert(er, IsNil)
+	// shouldn't this assert tests for result from line 284
```

Comment:
I think we can remove line285-286, and we can use `err` instead of `er`.

Reply:
After removing line285-286, `c.Assert(terror.ErrorEqual(err, ErrWrongValueForVar), IsTrue)` raises error, not sure if I should change to `IsFalse` or fix the test, since this line originally checks for err in line 281.

Reply:
I file #23017. I think here we can add a `// FIXME: issue #23017` here.

Reply:
`Fixme` added.